src/js/views/forms/form/form_views.js

```diff
@@ -44,9 +44,9 @@ const FormActionsView = View.extend({
   className: 'form__actions',
   template: hbs`
     {{#if hasHistory}}<button class="js-history-button form__actions-icon{{#if shouldShowHistory}} is-selected{{/if}}">{{far "history"}}</button>{{/if}}
```

Comment:
Looks like the `"history"` icon name was also changed to `"clock-rotate-left"` in v6 (via the [Icons Renamed In Version 6](https://fontawesome.com/docs/web/setup/upgrade/whats-changed#icons-renamed-in-version-6) list.).

There's one more instance of this on line 292 of this file.

Reply:
nice catch